.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "pwa-chrome",
+      "request": "launch",
+      "name": "Launch Chrome against localhost",
+      "url": "http://localhost:4242/umsoknir/faedingarorlof/",
+      "webRoot": "${workspaceFolder}/island.is"
+    }
+  ]
+}
```

Comment:
Please do not commit this file for now. Should probably update .gitignore so it is ignored until we want to standardise this file.